explainer.md

```diff
@@ -89,21 +89,29 @@ The second mode where styles are copied to a container element won't be part of
 
 ### Nested transition elements
 
-In the example transition, the content of the header cross-fades from Page-A to Page-B. An even smoother transition could be achieved by also animating the site title and avatar 'chip' independently. However, 'v1' of this proposal disallows a shared element to be nested inside another shared element.
+In the example transition, the content of the header cross-fades from Page-A to Page-B. An even smoother transition could be achieved by also animating the site title and avatar 'chip' independently. To allow for this, an offered element can contain other offered elements.
 
-The restriction avoids the need to preserve the hierarchy of shared elements and associated properties (transform, clip, effects inherited by descendants) in the DOM representation created to render the images referenced above. This helps in minimizing the scope for 'v1' of the feature.
+When an element is captured, its painting is 'lifted' from the parent offered element, which may be the root. This is similar to how browsers handle composited elements.
 
-- [Open question](https://github.com/WICG/shared-element-transitions/issues/74): If the developer tries to do this, what should happen, and when is that verified? Is the nested offering ignored? Is the whole transition abandoned?
+### Retaining hierarchy during a transition
```

Comment:
@khushalsagar I reworked this idea a bit so it feels less like a v2 switch, and more like opting into a new positioning space like `position: relative`. Let me know what you think. I went back and forth on elements of this, so I'm absolutely open to tweaks and better ideas.

Reply:
The idea sounds interesting. Do you have any example in mind that could make a case for your parent being different from the nearest ancestor shared element? And also for having a completely flat list.

Because the alternate choice we were debating was to have every offered element be a 'transition container'. With the assumption that there is no use-case where the developer would want to change this hierarchy or have a flat list. If we could point to a use-case then it would help with making an argument for the flat mode being a useful mode and not just a stop-gap until we have hierarchy.

Reply:
I tried to include a bit of this in the explainer. A flat list allows you to scale the parent without stretching the children, or needing to provide a counter-transform to provide that. A flat list also makes it easier if a child element moves containers between pages.